native/c/makefile

```diff
@@ -44,7 +44,6 @@ MACLIBS= -I../asmmac \
  -ICBC.SCCNSAM
 
 MTL_HEADERS=-I/usr/include/metal \
- -I/usr/include \
```

Comment:
I believe this was added to support `hwtjic.h` for JSON parsing because it is located in `/usr/include`.

However since we don't use the JSON parser in the backend code (only tests and middleware), I'm ok with leaving it removed for now 😋